app/forms/ConfidentialInformationFormProvider.scala

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright 2018 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import javax.inject.Inject
+
+import forms.mappings.Mappings
+import play.api.data.Form
+import play.api.data.Forms._
+import models.ConfidentialInformation
+
+class ConfidentialInformationFormProvider @Inject() extends Mappings {
+
+   def apply(): Form[ConfidentialInformation] = Form(
+     mapping(
+      "field1" -> text("confidentialInformation.error.field1.required")
```

Comment:
I would give a meaningful name to the fields.
Name `field1` is not great

Reply:
Again autogenerated by scaffold, but can be edited. Just takes time updating in every place it exists.

Reply:
I guess the problem is that field we don't change is a small bit of tech debt, which gradually accrues over time. It's a lot easier to change it as we go along, while we're adding each page, rather than doing huge tech debt fixes later on